packages/backend/server/src/core/doc/manager.ts

```diff
@@ -719,7 +755,8 @@ export class DocManager implements OnModuleInit, OnModuleDestroy {
     job: () => Promise<T>
   ) {
     return this.doWithLock(
-      `doc:manager:updates-lock:${workspaceId}::${guid}`,
+      'doc:manager:updates',
```

Comment:
can use `UPDATES_QUEUE_CACHE_KEY`

Reply:
this is the scope name of the lock, not the same stuff